DESCRIPTION

```diff
@@ -11,8 +11,8 @@ Description: The ReactomeGSA packages uses Reactome's online analysis service to
     Therefore, if you only require a gene set analysis, different packages are more suited.
 License: MIT + file LICENSE
 Encoding: UTF-8
+Imports: jsonlite, httr, progress, ggplot2, methods, gplots, RColorBrewer, dplyr, tidyr, SingleCellExperiment, scran, scater,  Seurat, scuttle
```

Comment:
`imports` defines packages that are needed for the functionality of the package. Meaning, if people want to install `ReactomeGSA` the other packages need to be installed as well.

`enhances` states that these packages profit from ReactomeGSA, but it works without them.

Therefore, I try to keep `Imports` as light as possible.

Here, I'd suggest to move SingleCellExperiment, scran, scater,  Seurat, scuttle to `enhances` as these packages are only needed for the scRNA-seq functions. Otherwise, the install process becomes very heavy.